controllers/cryptoController.js

```diff
@@ -0,0 +1,90 @@
+const usersdata = require('../models/crypto')
+const calculation = require('../middlewares/cryptoTransactionClac')
+/**
+ * Send Money from one User to Other
+ * @param {Object} req - Express request object
+ * @param {Object} res - Express response object
+ */
+const send = async (req, res) => {
+  try {
+    const request = req.body
+    const [userTo, userFrom, currency, amount] = [request.to, request.from, request.currency, request.amount]
+    const message1 = `${amount} ${currency} Coins Transfered From ${userFrom} to ${userTo}`
+    const verifyUserTO = await usersdata.fetchUser(userTo)
+    const verifyUserFrom = await usersdata.fetchUser(userFrom)
```

Comment:
use Promise.all api to fetch both things and then await

Reply:
ok working on it.